starters/next/components/Layout.js

```diff
@@ -1,22 +1,4 @@
-import { useCart, useCheckout } from '@nacelle/react-hooks';
-import { useEffect } from 'react';
-
-// This component utilizes `useCart` and `useCheckout` hooks from
-// `@nacelle/react-hooks` to clear cart and checkout data if the
-// checkout is completed.
-// https://github.com/getnacelle/nacelle-react/tree/main/packages/react-hooks
-
 function Layout({ children }) {
-  const [, { clearCart }] = useCart();
-  const [{ completed }, { clearCheckoutData }] = useCheckout();
-
-  useEffect(() => {
-    if (completed) {
-      clearCheckoutData();
-      clearCart();
-    }
-  }, [completed, clearCheckoutData, clearCart]);
```

Comment:
I can't remember what we decided re: knowing when the cart should be cleared when using Shopify Cart 🤔

Reply:
@NWRichmond  not clear on this one either to be honest

Reply:
We figured this out elsewhere, but are sharing here for posterity's sake:

After completing a checkout with a cart's `checkoutUrl`, that `cart` is returned as `null` from Shopify.

So when the page is refreshed or the site is loading for the first time, and a cart ID is persisted in browser storage, if the value of
```js
cartClient.cart({ cartId: 'the-persisted-cart-id' });
```
is `null`, we can clear the cart.